cmd/root.go

```diff
@@ -23,6 +23,6 @@ func Execute() {
 }
 
 func exitWithError(err error) {
-	fmt.Println(color.RedString(err.Error()))
+	fmt.Fprintf(color.Output, "%s\n", color.RedString(err.Error()))
```

Comment:
After looking at the [docs](https://pkg.go.dev/github.com/fatih/color#Error), `color.Error` should be used here